voidcrew/modules/cloning/code/machinery_cloning.dm

```diff
@@ -0,0 +1,572 @@
+		log_cloning("[key_name(mob_occupant)] destroyed within [src] at [AREACOORD(src)] due to malfunction.")
+		QDEL_IN(mob_occupant, 40)
+
+/obj/machinery/clonepod/relaymove(mob/user)
+	container_resist(user)
+
+/obj/machinery/clonepod/container_resist(mob/living/user)
+	if(user.stat == CONSCIOUS)
+		go_out()
+
+/obj/machinery/clonepod/emp_act(severity)
+	. = ..()
+	if (!(. & EMP_PROTECT_SELF))
+		var/mob/living/mob_occupant = occupant
+		if(mob_occupant && prob(100/(severity*efficiency)))
+			connected_message(Gibberish("EMP-caused Accidental Ejection"))
+			SPEAK(Gibberish("Exposure to electromagnetic fields has caused the ejection of [mob_occupant.real_name] prematurely."))
+			go_out()
+			log_cloning("[key_name(mob_occupant)] ejected from [src] at [AREACOORD(src)] due to EMP pulse.")
+
+/obj/machinery/clonepod/ex_act(severity, target)
+	..()
+	if(!QDELETED(src) && occupant)
+		var/mob/living/mob_occupant = occupant
+		go_out()
+		log_cloning("[key_name(mob_occupant)] ejected from [src] at [AREACOORD(src)] due to explosion.")
+
+/obj/machinery/clonepod/handle_atom_del(atom/A)
+	if(A == occupant)
+		occupant = null
+		countdown.stop()
+
+/obj/machinery/clonepod/proc/horrifyingsound()
+	for(var/i in 1 to 5)
+		playsound(src,pick('sound/hallucinations/growl1.ogg','sound/hallucinations/growl2.ogg','sound/hallucinations/growl3.ogg'), 100, (rand(95,105) * 0.01))
+		sleep(1)
+	sleep(10)
+	playsound(src,'sound/hallucinations/wail.ogg', 100, TRUE)
+
+/obj/machinery/clonepod/deconstruct(disassembled = TRUE)
+	if(occupant)
+		var/mob/living/mob_occupant = occupant
+		go_out()
+		log_cloning("[key_name(mob_occupant)] ejected from [src] at [AREACOORD(src)] due to deconstruction.")
+	..()
+
+/obj/machinery/clonepod/proc/maim_clone(mob/living/carbon/human/H)
+	if(!unattached_flesh)
+		unattached_flesh = list()
+	else
+		for(var/fl in unattached_flesh)
+			qdel(fl)
+		unattached_flesh.Cut()
+
+	H.setCloneLoss(CLONE_INITIAL_DAMAGE)     //Yeah, clones start with very low health, not with random, because why would they start with random health
+	// In addition to being cellularly damaged, they also have no limbs or internal organs.
+	// Applying brainloss is done when the clone leaves the pod, so application of traumas can happen
+	// based on the level of damage sustained.
+
+	if(!HAS_TRAIT(H, TRAIT_NODISMEMBER))
+		var/static/list/zones = list(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)
+		for(var/zone in zones)
+			var/obj/item/bodypart/BP = H.get_bodypart(zone)
+			if(BP)
+				BP.drop_limb()
+				BP.forceMove(src)
+				unattached_flesh += BP
+
+	for(var/o in H.internal_organs)
+		var/obj/item/organ/organ = o
+		if(!istype(organ) || (organ.organ_flags & ORGAN_VITAL))
+			continue
+		organ.organ_flags |= ORGAN_FROZEN
+		organ.Remove(H, special=TRUE)
+		organ.forceMove(src)
+		unattached_flesh += organ
+
+	flesh_number = unattached_flesh.len
+
+/*
+ *	Manual -- A big ol' manual.
+ */
+
+/obj/item/paper/guides/jobs/medical/cloning
```

Comment:
Remove this, I don't want to deal with paper changes in the future and I'm pretty sure cloning is pretty self-explanatory, and I don't think it's possible for players to get it anyways.